frontend/luasettings.lua

```diff
@@ -28,6 +28,38 @@ function LuaSettings:delSetting(key)
     self.data[key] = nil
 end
 
+function LuaSettings:has(key)
+    return self:readSetting(key) ~= nil
+end
+
+function LuaSettings:hasNot(key)
+    return self:readSetting(key) == nil
+end
+
+function LuaSettings:True(key)
```

Comment:
I suggest we rename this function to `isTrue` or `truthy` instead.

Another question, what happens if `self:readSetting(key)` returns a boolean value?